org.eclipse.jdt.core.manipulation/META-INF/MANIFEST.MF

```diff
@@ -49,7 +50,11 @@ Export-Package: org.eclipse.jdt.core.manipulation,
  org.eclipse.jdt.internal.corext.refactoring.structure;x-friends:="org.eclipse.jdt.ui",
  org.eclipse.jdt.internal.corext.refactoring.surround;x-friends:="org.eclipse.jdt.ui",
  org.eclipse.jdt.internal.corext.refactoring.tagging;x-friends:="org.eclipse.jdt.ui",
- org.eclipse.jdt.internal.corext.refactoring.typeconstraints.types;manipulation=split;mandatory:=manipulation;x-friends:="org.eclipse.jdt.ui",
+ org.eclipse.jdt.internal.corext.refactoring.generics;x-friends:="org.eclipse.jdt.ui",
```

Comment:
Ugh, just noticed this. Missing the `manipulation=split;mandatory:=manipulation`. I would preserve that.

Reply:
shoot don't merge yet it I did it wrong

Reply:
okay. Should be fixed